packages/yak-swc/yak_swc/src/utils/css_hash.rs

```diff
@@ -0,0 +1,106 @@
+use std::hash::{DefaultHasher, Hash, Hasher};
+
+/// Creates a CSS-compatible identifier using Rust's DefaultHasher (SipHash-1-3)
+///
+/// Takes an ASCII string and returns a base62 encoded hash result
+/// that is a valid CSS identifier starting with a letter
+pub fn hash_to_css(input: &str) -> String {
+  let mut hasher = DefaultHasher::new();
+  input.hash(&mut hasher);
+  let hash = hasher.finish();
+  to_css_identifier(hash)
+}
+
+// Helper function to convert u64 to a valid CSS identifier
+fn to_css_identifier(mut num: u64) -> String {
+  // CSS Identifiers must start with a letter
+  const FIRST_LETTER_DIGITS: &[u8] = b"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  let first_letter_count = FIRST_LETTER_DIGITS.len() as u64;
+  // CSS Identifiers can contain letters, digits, hyphens, and underscores
+  const DIGITS: &[u8] = b"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+  let digit_count = DIGITS.len() as u64;
+
+  let mut result = Vec::new();
```

Comment:
```suggestion
  let mut result = Vec::with_capacity(6);
```

And maybe add another constant for the 6 char length?